src/coreclr/jit/llvm.cpp

```diff
@@ -1487,12 +1452,23 @@ void storeLocalVar(llvm::IRBuilder<>& builder, GenTreeLclVar* lclVar)
     }
 }
 
+void replaceMappedLlvmValueWithShadowStack(GenTree* node)
+{
+    _sdsuMap->at(node) = LocatedLlvmValue(ValueLocation::LlvmStack, _function->getArg(0));
+}
```

Comment:
Ditto: the arguments & shadow stack.

What prevents us from, when seeing the shadow stack local node in the IR, map it to the shadow stack arg?